experimental/web_dashboard/lib/src/pages/item_details.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:flutter/material.dart';
```

Comment:
nit: copyright header.

Reply:
good catch. We need a tool that can check this...

Reply:
Wouldn't be a hard thing to write. We have the /tool directory for those sorts of scripts now, too.